corehq/apps/reports/v2/models.py

```diff
@@ -130,3 +132,14 @@ def get_context(self):
         :return: {}
         """
         raise NotImplementedError("please implement get_context")
+
+
+class BaseFilter(object):
```

Comment:
If this base class just has overrideable methods, might be worth making it an [Abstract Base Class](https://docs.python.org/3/library/abc.html)

Reply:
updated thanks!